gematria/experiments/access_pattern_bm/BUILD

```diff
@@ -0,0 +1,95 @@
+package(
+    default_visibility = ["//visibility:private"],
+)
+
+cc_library(
+    name = "linked_list",
+    srcs = ["linked_list.cc"],
+    hdrs = ["linked_list.h"],
+    copts = ["-mclflushopt"],
+    visibility = ["//:internal_users"],
+)
+
+cc_test(
+    name = "linked_list_bm",
+    size = "small",
+    srcs = ["linked_list_bm.cc"],
+    deps = [
+        ":linked_list",
+        "@com_google_benchmark//:benchmark",
+    ],
+)
```

Comment:
I don't quite expect the individual cc_libraries to be re-used elsewhere, but left them separate considering that maybe the need arises to have some other variation of the microbenchmark using the same code but handling things somewhat differently. Would it be better to have them be merged now, but split up later in case it becomes a requirement?